setup/TemplateConfigurator.rb

```diff
@@ -70,7 +70,7 @@ def ask_with_answers(question, possible_answers)
     def run
       @message_bank.welcome_message
 
-      framework = self.ask_with_answers("What language do you want to use?", ["ObjC", "Swift"]).to_sym
+      framework = self.ask_with_answers("What language do you want to use?", ["Swift", "ObjC"]).to_sym
```

Comment:
Interesting that the only real problem is with dependent Pods. If we added an ability to define the swift version in the podfile / podspec this case wouldn't exist.